packages/orbit-components/src/Modal/ModalFooter/__tests__/index.test.tsx

```diff
@@ -20,6 +20,16 @@ describe("ModalFooter", () => {
     });
   });
 
+  it("should not have StyledChild wrapper on single children", () => {
+    render(
+      <ModalFooter dataTest="footer">
+        <Button type="secondary">Cancel</Button>
+      </ModalFooter>,
+    );
+
+    expect(screen.findByTestId("footer-el-wrapper")).toMatchObject({});
```

Comment:
Just for consistency with other tests we have around the repo, why don't we do it like this?

```suggestion
    expect(screen.queryByTestId("footer-el-wrapper")).not.toBeInTheDocument();
```

Reply:
It won't work for that case